mobile/src/main/java/ch/epfl/sdp/mobile/application/tournaments/StoreDocumentTournament.kt

```diff
@@ -129,7 +133,7 @@ class StoreDocumentTournament(
       val round = document.stage?.toIntOrNull() ?: 1
       val results =
           store
-              .collection("games")
+              .collection(ChessDocument.Collection)
               .whereEquals("tournamentId", reference.uid)
               .whereEquals("roundDepth", round)
```

Comment:
Missing field.

Reply:
Added in 715a39794e4fd9228eaaf11ad0bf8a1682f5edfd